### DISCUSSION TITLE
Feature: show more information of current disk in label

### DISCUSSION THREAD

**Discussion by itlezy** [doublecmd/doublecmd#1178] [Suggestions]
Hi,
Thanks for this great software.

I would like the option to show show more information of the current disk directly in the label above the file list.
i.e.
`d: [DISK LABEL] 123k of 999k free`

Thanks

**Comment by avmaksimov** [comment]
I used to see this information with the similar file manager, but after I found out "Drive list button" and may be it would be enough for you too?
